app/controllers/static_pages_controller.rb

```diff
@@ -7,6 +7,7 @@ def show
     # static pages
     @sitehome = AdoptionHomePage.instance
     @adopteve = BookAdoptionEvePage.instance
+    @option = RootSettingsPage.instance
```

Comment:
Is `@option` where the root settings are stored? If so could we perhaps rename it `@rootSettings` so that's more obvious? I might have misunderstood what it's for so just let me know if I have 😄

Reply:
yes! Option was the name of the table haha